### ISSUE TITLE
MobX 6.X Usage

### ISSUE THREAD

**Issue by ragecryx** [pinqy520/mobx-persist#88]
Can the library be used with MobX version 6.0? I can't seem to find an example using `makeObservable` function.

**Comment by paramoNNNN** [comment]
Same, I can't make it work with makeObservable.

**Comment by idanlevi1** [comment]
Any solution?

**Comment by idanlevi1** [comment]
> Any solution?

After a while, my solution: 

### MobX 6 starter - with mobx-persist-store & without decorators
  [Github Source](https://github.com/idanlevi1/mobx6-with-persist-starter) / [Code Sendbox](https://codesandbox.io/s/mobx6-with-persist-starter-uq818)

**Comment by eladgel** [comment]
@idanlevi1 so different library?

**Comment by idanlevi1** [comment]
> @idanlevi1 so different library?

@eladgel Yes, we gave up on `mobx-persist`, the package did not update for 3 years so we change to a new one
I wrote a guide for it:
[https://idanlevi2.medium.com/react-with-mobx-6-with-persist-and-without-decorators-bd7e5a0578d6](https://idanlevi2.medium.com/react-with-mobx-6-with-persist-and-without-decorators-bd7e5a0578d6)

And if you want more details you can send me a message (also in Hebrew 😉)

**Comment by eladgel** [comment]
@idanlevi1 That's great. We use React-Native and the Wix RNN, So we will need to migrate to that as well
